packages/ocamlfuse/ocamlfuse.2.7.1-cvs8/opam

```diff
@@ -0,0 +1,51 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "OCaml bindings for FUSE (Filesystem in UserSpacE)"
+description: """
+This is a binding to FUSE for the OCaml programming language, enabling
+you to write multithreaded filesystems in the OCaml language. It has
+been designed with simplicity as a goal, as you can see by looking at
+example/fusexmp.ml. Efficiency has also been a separate goal. The
+Bigarray library is used for read and writes, allowing the library to
+do zero-copy in OCaml land."""
+maintainer: ["Alessandro Strada <alessandro.strada@gmail.com>"]
+authors: [
+  "Vincenzo Ciancia"
+  "Olaf Hering <olaf@aepfle.de>"
+  "Alessandro Strada <alessandro.strada@gmail.com>"
+]
+license: "GPL-1.0-or-later"
+bug-reports: "https://github.com/astrada/ocamlfuse/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "ocaml" {>= "4.02.3"}
+  "base-bigarray"
+  "base-threads"
+  "base-unix"
+  "camlidl"
+  "dune-configurator"
+  "conf-libfuse"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+available: [os != "macos"]
```

Comment:
```suggestion
```
In this case I suggest you let camlidl fail (I will fix camlidl opam file separately) and when a new version is released, if the CI fails, I will manually mark this as unavailable on macos